hostinfo/subscription.go

```diff
@@ -60,7 +60,7 @@ func GetSocketCount(facts SubManValues) (string, error) {
 }
 
 func GetProduct(facts SubManValues) ([]string, error) {
-	output, _ := execSubManCommand("list --installed")
+	output, _ := execSubManCommand("list", "--installed")
```

Comment:
Is this worthy of a test to ensure the command gets called correctly?

Reply:
I've modified the `mocks/subscription-manager` to fail if wrong command is passed thus it is noticeable in `make test-daemon` (though it will not fail, one must inspect the output) - retrospectively the wrong mock was an indicator of the bug.

Doing it properly is much more effort and actually a content of https://issues.redhat.com/browse/HMS-3022